pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -83,17 +85,17 @@ func writeVifFile(buf []byte, pid, name string) error {
 	return nil
 }
 
-func setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, uid string) error {
+func (l *podNICImpl) setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, uid string) error {
```

Comment:
If you're moving these functions to be methods of the `podNICImpl` struct, shouldn't you also set iface / podInterfaceName / UID as members of the struct ? 

That way, you can clean the method's signature.

Reply:
I am doing that at next PR that remove the pidfactory alltogether https://github.com/kubevirt/kubevirt/pull/5062/commits/dd68d660535f9802bd2bcb9ef2a283d8ab2fa7fe#diff-030aa8b7ff9bf4a511d1d5641c1dec04205a944137672be3a691f0b1b4068753R87-R92